### ISSUE TITLE
[Discuss] Navbar spacing fix and refactor

### ISSUE THREAD

**Issue by austinogiza** [layer5io/layer5#1788]
**Current Behavior**
The current navbar has inconsistent spacing and looks choked

**Desired Situation**
We should try achieving consistent spacing in the discuss.ayer5.io navbar same as with the design.
<img width="1440" alt="discuss navbar" src="https://user-images.githubusercontent.com/44683018/114949503-13fbc380-9e49-11eb-9f96-dc239d2f0cd7.png">

Here is the figma link to the navbar frame
https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs?node-id=5398%3A0

**Comment by imraghav20** [comment]
Hi @austinogiza 
I would like to take up this issue.

Are you talking about the navbar in [layer5.io](https://layer5.io/) website? Or some other website?
[I am unable to access discuss.layer5.io]

**Comment by leecalcote** [comment]
Excellent!

**Comment by Nikhil-Ladha** [comment]
The spacing issue is from the discourse side itself. It doesn't provide a facility to Center the navbar.
Do check with @Jashpatel1 once about the possibilities of the same.

**Comment by imraghav20** [comment]
I have customised the header to this
![curr navbar](https://user-images.githubusercontent.com/59257038/115022012-6638f500-9eda-11eb-9ca1-1b68e0f9c820.PNG)


Earlier it was like this
![prev navbar](https://user-images.githubusercontent.com/59257038/115021819-1ce8a580-9eda-11eb-9a6f-61f541a113e8.PNG)

**Comment by imraghav20** [comment]
Should I even move `Sign Up` and `Log In` buttons to right of search and menu?

**Comment by Jashpatel1** [comment]
> Should I even try to move `Sign Up` and `Log In` buttons to right of search and menu?

Yes, please go ahead.

**Comment by imraghav20** [comment]
I explored `Discourse` but could not find any solution.
It has option to edit only the CSS for header but not HTML.

If we write any HTML inside the `Header` tab of Discourse, then one more header is added at the top.
So, I created a new header in the `Header` tab and set `display: none` for the pre-built header.
This solution worked as far as the UI is concerned but the `Log in` and `Sign up` button lost their functionalities.
